platform/x11/cog-platform-x11.c

```diff
@@ -432,11 +434,19 @@ xcb_process_events (void)
         case XCB_VISIBILITY_NOTIFY:
             xcb_handle_visibility_event((xcb_visibility_notify_event_t *) event);
             break;
+        case XCB_EXPOSE: {
```

Comment:
Is the style incorrect on the previous case events, or should the brace go on the next line here?

Reply:
This is the correct way, the previous ones are wrong; but the style checker runs only on the changed parts of the code, that's why the other ones are not reported as being wrong.